src/python/turicreate/test/test_boosted_trees_checkpoint.py

```diff
@@ -12,8 +12,10 @@
 import tempfile
 import os
 import shutil
+import pytest
 
 
+@pytest.mark.minimal
```

Comment:
I don't think this test should be included.

Reply:
Right! Good catch!